cluster/gce/gci/configure-helper.sh

```diff
@@ -626,6 +635,8 @@ function prepare-etcd-manifest {
   sed -i -e "s@{{ *etcd_cluster *}}@$etcd_cluster@g" "${temp_file}"
   sed -i -e "s@{{ *storage_backend *}}@${STORAGE_BACKEND:-}@g" "${temp_file}"
   sed -i -e "s@{{ *cluster_state *}}@$cluster_state@g" "${temp_file}"
+  sed -i -e "s@{{ *etcd_protocol *}}@https@g" "${temp_file}"
```

Comment:
It seems that with GCI we are always using https and it's not possible to use http. Is that what we want?
Do we have any performance results for this change?